AMBuildScript

```diff
@@ -369,11 +395,42 @@ class AMXXConfig(object):
     binary = context.compiler.Program(name)
     return self.AddVersioning(binary)
 
+  def AddAssembly(self, context, binary, input_file, output_file, includes=[], extra_argv=[]):
+    if builder.target_platform == 'windows':
+      obj_type = 'win32'
+    elif builder.target_platform == 'linux':
+      obj_type = 'elf'
```

Comment:
Maybe have it as `elf32` as well to match the other two, or remove `32` from the other two. Is there a reason for this one to be different?